generated/nirfmxbluetooth/nirfmxbluetooth.proto

```diff
@@ -0,0 +1,2025 @@
+
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Proto file for the NI-RFMXBT Metadata
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ni.grpc.nirfmxbt";
+option java_outer_classname = "NiRFmxBT";
```

Comment:
What's our capitalized name? `Bluetooth`?

Reply:
Yes, and these package names are fixed now.